benchmark/src/main/java/com/linecorp/decaton/benchmark/TemporaryTopic.java

```diff
@@ -41,7 +41,7 @@ public static TemporaryTopic create(String bootstrapServers, String topic) {
     @Override
     public void close() throws Exception {
         log.info("Cleaning up topic {}", topic);
-        admin.deleteTopics(topic);
+        admin.deleteTopics(true, topic);
```

Comment:
I think this should be false because benchmark could run against a custom cluster (that could be an actually servicing cluster) depending on configuration rather than a local embedded cluster and in case we must clean it up.